packages/indexer-agent/src/commands/start.ts

```diff
@@ -427,6 +444,7 @@ export async function createNetworkSpecification(
       subgraphs,
       networkProvider,
       addressBook: argv.addressBook,
+      escrowAddressBook,
```

Comment:
Can't we add the escrow addresses into the original `addressBook` instead of adding all of that code?

Reply:
it was done in this way so that if necesary you can pass the values for the escrowAddress, else it will take those it has already. (this was done overall to make testing easier)